landing_page/templates/landing_page/search.html

```diff
@@ -12,9 +12,9 @@
         {% if movies %}
             {% for movie in movies %}
                 <div class="search_result" id="movie_{{ movie.id }}">
-                    <img loading="lazy" id="poster_{{ movie.id }}" class="poster" alt="{{ movie.name }}" onclick="loadDetailsPage('{{ movie }}')" src="https://image.tmdb.org/t/p/w200{{movie.poster}}">
+                    <img loading="lazy" id="poster_{{ movie.id }}" class="poster" alt="{{ movie.name }}" onclick="loadDetailsPage({{ movie.id }})" src="https://image.tmdb.org/t/p/w200{{movie.poster}}">
                     <div class="searched_movie_details">
-                        <h1 class="movie_title"><a class="movie_link" href="/movie?movie=${encodeURIComponent(JSON.stringify({{movie}}))}">{{ movie.name }}</a></h1>
+                        <h1 class="movie_title"><a class="movie_link" href="/movie/{{movie.id}}">{{ movie.name }}</a></h1>
```

Comment:
this also doesn't work :(
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/ca345642-cb14-42ac-9f2f-dc1779400499)